goon/browserassets/css/browserOutput-dark.css

```
@@ -242,162 +242,161 @@ a.popt {text-decoration: none;}
-.slime					{color: #0077AA;}
-.drask					{color: #a3d4eb; font-family: "Arial Black";}
-.moth					{color: #869b29; font-family: "Copperplate";}
-.clown					{color: #ff0000;}
+.kidan				{color: #C64C05;}
+.slime				{color: #0077AA;}
+.drask				{color: #a3d4eb; font-family: "Arial Black";}
+.moth				{color: #869b29; font-family: "Copperplate";}
+.clown				{color: #ff0000;}
 .vulpkanin				{color: #B97A57;}
 .abductor				{color: #800080; font-style: italic;}
 .mind_control			{color: #A00D6F; font-size: 3; font-weight: bold; font-style: italic;}
-.rough					{font-family: 'Trebuchet MS', cursive, sans-serif;}
-.say_quote				{font-family: Georgia, Verdana, sans-serif;}
-.cult					{color: #800080; font-weight: bold; font-style: italic;}
-.cultspeech				{color: #AF0000; font-style: italic;}
+.rough				{font-family: 'Trebuchet MS', cursive, sans-serif;}
+.say_quote			{font-family: Georgia, Verdana, sans-serif;}
+.cult				{color: #800080; font-weight: bold; font-style: italic;}
+.cultspeech			{color: #AF0000; font-style: italic;}
 .cultitalic				{color: #A60000; font-style: italic;}
 .cultlarge				{color: #A60000; font-weight: bold; font-size: 120%;}
-.narsie					{color: #A60000; font-weight: bold; font-size: 300%;}
+.narsie				{color: #A60000; font-weight: bold; font-size: 300%;}
 .narsiesmall			{color: #A60000; font-weight: bold; font-size: 200%;}
 .interface				{color: #9031C4;}
-.big					{font-size: 150%;}
+.big				{font-size: 150%;}
 .reallybig				{font-size: 175%;}
-.greentext				{color: #00FF00;	font-size: 150%;}
+.greentext			{color: #00FF00;	font-size: 150%;}
 .redtext				{color: #FF0000;	font-size: 150%;}
-.bold					{font-weight: bold;}
-.his_grace 				{color: #15D512; font-family: "Courier New", cursive, sans-serif; font-style: italic;}
-.center					{text-align: center;}
-.red					{color: #FF0000;}
-.purple					{color: #9031C4;}
+.bold				{font-weight: bold;}
+.his_grace 			{color: #15D512; font-family: "Courier New", cursive, sans-serif; font-style: italic;}
+.center				{text-align: center;}
+.red				{color: #FF0000;}
+.purple				{color: #9031C4;}
 .skeleton				{color: #C8C8C8; font-weight: bold; font-style: italic;}
-.gutter					{color: #7092BE; font-family: "Trebuchet MS", cursive, sans-serif;}
-.orange					{color: #ffa500;}
+.gutter				{color: #7092BE; font-family: "Trebuchet MS", cursive, sans-serif;}
+.orange				{color: #ffa500;}
 .orangei				{color: #ffa500; font-style: italic;}
 .orangeb				{color: #ffa500; font-weight: bold;}
 .resonate				{color: #298F85;}
 
-.revennotice				{color: #6685F5;}
+.revennotice			{color: #6685F5;}
 .revenboldnotice			{color: #6685F5;	font-weight: bold;}
-.revenbignotice				{color: #6685F5;	font-weight: bold; font-size: 120%;}
-.revenminor				{color: #823abb}
-.revenwarning				{color: #760fbb;	font-style: italic;}
-.revendanger				{color: #760fbb;	font-weight: bold; font-size: 120%;}
+.revenbignotice			{color: #6685F5;	font-weight: bold; font-size: 120%;}
+.revenminor			{color: #823abb}
```

```suggestion
.revenminor			{color: #823abb;}
```
inconsistent attribute ending